django/publicmapping/redistricting/templates/editplan.html

```diff
@@ -458,6 +458,10 @@ <h2>{% blocktrans %}3. Paste selected {{ body_member_long_label }} into working
                   </li>
                 </ul></td>
                 <td><ul>
+                  <li>
+                    <input name="values" id="val_incumbent" class="values_choices" value="Incumbent Protection" type="checkbox" />
+                    <label for="val_adv">{% trans "Incumbent Protection" %}</label>
```

Comment:
This should be `val_incumbent`, not `val_adv`. As it is currently, clicking on this label will check the wrong box.